Library/Formula/sshuttle.rb

```diff
@@ -11,7 +11,10 @@ def install
     # Building the docs requires installing
     # markdown & BeautifulSoup Python modules
     # so we don't.
-    libexec.install Dir["*"]
-    bin.write_exec_script libexec/"src/sshuttle"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
```

Comment:
`sshuttle` was always written in Python. It recently went through some code restructuring in the 0.73 release where using the original `homebrew` install no longer worked.